src/content/docs/apm/agents/net-agent/other-installation/understanding-net-agent-environment-variables.mdx

```diff
@@ -0,0 +1,217 @@
+
+#### For .NET framework applications:
+
+`COR_ENABLE_PROFILING` tells the runtime whether or not to enable profiling for an application. Set to 1 to enable profiling, set to 0 to disable it.
+
+`COR_PROFILER` tells the runtime which profiler to use. The value is a GUID unique to a given profiler. For New Relic's profiler for .NET Framework, the correct value is `{71DA0A04-7777-4EC6-9643-7D28B46A8A41}`.
+
+`COR_PROFILER_PATH` tells the runtime where to find the profiler on the system, and depends on the install type and platform. Note that this may not be required if the agent was installed with the MSI installer, which registers the profiler DLL with the system based on the GUID set in `COR_PROFILER`. However, it's safe to have it set anyway as long as the value is set to the correct path.
+
+Typical values:
+
+```
+COR_ENABLE_PROFILING=1
+COR_PROFILER={71DA0A04-7777-4EC6-9643-7D28B46A8A41}
+COR_PROFILER_PATH="C:\Program Files\New Relic\.NET Agent\netframework\NewRelic.Profiler.dll"
+```
+
+#### For .NET Core applications:
+
+`CORECLR_ENABLE_PROFILING` tells the runtime whether or not to enable profiling for an application. Set to 1 to enable profiling, set to 0 to disable it.
+
+`CORECLR_PROFILER` tells the runtime which profiler to use. The value is a GUID unique to a given profiler. For New Relic's profiler for .NET Core, the correct value is `{36032161-FFC0-4B61-B559-F6C5D41BAE5A}`.
+
+`CORECLR_PROFILER_PATH` tells the runtime where to find the profiler on the system.
+
+Typical values:
+
+```
+CORECLR_ENABLE_PROFILING=1
+CORECLR_PROFILER={36032161-FFC0-4B61-B559-F6C5D41BAE5A}
+CORECLR_PROFILER_PATH="C:\Program Files\New Relic\.NET Agent\netcore\NewRelic.Profiler.dll"
+```
+
+Note: There is a subtle difference in the behavior of the .NET framework agent vs. the .NET core agent regarding having profiling enabled. For the .NET framework agent, even when profiling is enabled in the environment of a .NET framework process, it will only be instrumented by the agent if one of the following conditions are met:
+
+* The application is hosted in IIS and
+  * application pools are configured to be instrumented by default, or
+  * the application pool hosting the application is configured to be instrumented explicitly
+* The application is explicitly configured to be instrumented in the system-wide `newrelic.config` file
+* AgentEnabled="true" is set in an app-local `newrelic.config` file
+
+However, for the .NET core agent, any process where `CORECLR_ENABLE_PROFILING` is set to 1 in its environment will be instrumented by the agent. Therefore, we don't recommend setting this environment variable to 1 system-wide, as it may result in excess system resource consumption and/or more data being sent to New Relic than expected. Instead, only set this variable to 1 in the environment of the processes you want instrumented.
```

Comment:
```suggestion
However, for the .NET core agent, any process where `CORECLR_ENABLE_PROFILING` is set to 1 in its environment will be instrumented by the agent. Therefore, we don't recommend setting this environment variable to `1` system wide, as it may result in excess system resource consumption and/or more data being sent to New Relic than expected. Instead, only set this variable to `1` in the environment of the processes you want instrumented.
```